src/button/tests/functional/Button.ts

```diff
@@ -0,0 +1,73 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+
+function getPage(remote: any) {
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=button')
+		.setFindTimeout(5000);
+}
+
+registerSuite({
+	name: 'Button',
+
+	'button should be visible'(this: any) {
+		return getPage(this.remote)
+			.findByCssSelector('button:first-of-type')
```

Comment:
Thanks for the review Tom. I looked at https://github.com/dojo/widgets/pull/166/commits/6612379717688a6b3300fccd97d2546c32233846 where we changed to use `css-module` class to access the internals of a widget, and this approach makes sense to me. However, in my case here where the widget structure is simple and I am not trying to exercise the internal components of the widget, but the widget itself - I am trying to locate the first widget on the page (there's several button widgets in the example), so I am not sure if `css-module` class would be any help locating what I am looking for. For example, by using `css-module`, it seems I still have to do something like this: ``.findByCssSelector(`button.${css.root}`:first-of-type)``. That said, the example page probably should have included ids for each widget for external tests like this one.